components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/SyncLoginsClient.kt

```diff
@@ -0,0 +1,62 @@
+package mozilla.components.service.sync.logins
+
+import mozilla.appservices.logins.ServerPassword
+import mozilla.appservices.logins.SyncUnlockInfo
+
+/**
+ * Entrypoint for integrating with Firefox Sync for logins.
```

Comment:
Probably it would be better to indicate that this is a helper class. Users who don't need its functionality are free to ignore it.

Just changing "Entrypoint" to "Helper" is probably enough.